src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -9,191 +9,73 @@ object Arisa : ConfigSpec() {
-                """.trimIndent()
-            )
-            val moddedMessage by optional(
-                """
-                *Thank you for your report!*
-                However, this issue is {color:#FF5722}*Invalid*{color}.
-                
-                Your game, launcher or server is modified.
-                If you can reproduce the issue in a vanilla environment, please recreate the issue.
-                
-                * Any non-standard client/server/launcher build needs to be taken up with the appropriate team, not Mojang.
-                * Any plugin issues need to be addressed to the creator of the plugin or resource pack.
-                * If you have problems on large servers, such as The Hive and Hypixel, please contact them first as they run modified server software.
-                
-                *Quick Links*:
-                📓 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]
-                
-                ~{color:#888}-- I am a bot. This action was performed automagically! Please report any issues in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/]{color}~
-                """.trimIndent()
-            )
-
-            val duplicates by optional(
-                listOf(
-                    CrashDupeConfig("minecraft", "Pixel format not accelerated", "MC-297"),
-                    CrashDupeConfig("minecraft", "No OpenGL context found in the current thread", "MC-297"),
-                    CrashDupeConfig("minecraft", "Could not create context", "MC-297"),
-                    CrashDupeConfig("minecraft", "WGL: The driver does not appear to support OpenGL", "MC-128302"),
-                    CrashDupeConfig("minecraft", "failed to create a child event loop", "MC-34749"),
-                    CrashDupeConfig("minecraft", "Failed to check session lock, aborting", "MC-10167"),
-                    CrashDupeConfig("minecraft", "Maybe try a lowerresolution texturepack", "MC-29565"),
-                    CrashDupeConfig("minecraft", "java\\.lang\\.OutOfMemoryError\\: Java heap space", "MC-12949"),
-                    CrashDupeConfig("minecraft", "try a lowerresolution", "MC-29565"),
-                    CrashDupeConfig("java", "ig[0-9]{1,2}icd[0-9]{2}\\.dll", "MC-32606")
-                )
-            )
+            val maxAttachmentAge by optional(0, description = "Max age in days the attachment can have to be considered")
+            val crashExtensions by optional(emptyList<String>(), description = "File extensions that should be checked for crash reports.")
+            val duplicateMessage by optional("", description = "Message to be send when resolving a duplicate. {DUPLICATE} will be replaced by the ticket key")
```

Comment:
Instead of {DUPLICATE} you can use %s

Then on the usage code you do duplicateMessage.format(ticketKey) and you get the final string :D

Reply:
I think I copied that from the js version when I added the crash module :D